x-pack/plugins/security_solution/public/timelines/containers/all/index.tsx

```diff
@@ -132,13 +132,15 @@ export const useGetAllTimeline = (): AllTimelinesArgs => {
             loading: true,
           }));
 
-          const variables: GetAllTimelineVariables = {
-            onlyUserFavorite,
-            pageInfo,
+          const variables: GetTimelinesRequestQuery = {
+            only_user_favorite: onlyUserFavorite ? 'true' : 'false',
```

Comment:
This sadly won't work since the type only allows `true` and `false` and not arbitrary strings. `Boolean.toString()` returns `string` instead of `"true" | "false"` in TypeScript.